ui/components/ui/tabs/tabs.test.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { render, fireEvent } from '@testing-library/react';
-import Tabs from './tabs.component';
-import Tab from './tab/tab.component';
+import { Tabs } from './tabs';
+import { Tab } from './tab/tab';
```

Comment:
Updating Tabs and Tab to named exports, they are exported from index.ts as named so we don't have to change import paths outside of the tabs component